config.go

```diff
@@ -25,6 +29,15 @@ func LoadConfigFromPath(path string) (Config, error) {
 	if dsn, found := os.LookupEnv("TIGERBLOOD_DSN"); found {
 		config.DatabaseDsn = dsn
 	}
+	if dbMaxOpenConns, found := os.LookupEnv("TIGERBLOOD_DB_MAX_OPEN_CONNS"); found {
+		dbMaxOpenConns, err := strconv.Atoi(dbMaxOpenConns)
+		if err != nil {
+			config.DatabaseMaxOpenConns = dbMaxOpenConns
```

Comment:
Since it happens on launch time, I think it's better to fail. Admins are likely to start this as a service and assume that their configuration was accepted, if everything starts and works as expected.